### PR TITLE
Added new actions tests in Digital Transformation

### PR DESCRIPTION
**Pull request by KotliarovaK**

### CODE REVIEW

File: ui_tests_playwright/automation/test/ui/mainSite/services/digitalTransformation/DigitalTransformationActions.test.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import Buttons from '../../../../../identifiers/Buttons';
+import { ClutchReviewLinks } from '../../../../../preconditionsData/links/ClutchReviewLinks';
+
+test.beforeEach(async () => {
+	await baseDriverSteps.createsNewBrowserAndGoToUrl(UrlProvider.urlBuilder(UrlPath.DigitalTransform));
+});
+
+test('Check navigation to "Get in Touch" container after clicking CTA buttons from the "Digital Transformation" page @Regression @DigitalTransformation @TSWEB-1135', async () => {
+	const ctaButtons = [
+		driver.getByTestId(DigitalTransformation.Info).getByTestId(MainSiteButtons.RequestAQuote),
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/mainSite/services/digitalTransformation/DigitalTransformationActions.test.ts:21]: looks like it's not RequestAQuote button)

File: ui_tests_playwright/automation/test/ui/mainSite/services/digitalTransformation/DigitalTransformationActions.test.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+test('Check redirects by arrows in "Technologies We Use for Digital Transformation" container from the "Digital Transformation" page @Regression @DigitalTransformation @TSWEB-1135', async () => {
+	const technologiesContainer = driver.getByTestId(DigitalTransformation.TechnologiesWeUse);
+	const containerSection = technologiesContainer.getByTestId(Container.ContainerSection);
+	const arrows = containerSection.getByTestId(Container.Arrow);
+
+	const arrowUrlMap = new Map([
+		[arrows.nth(0), UrlProvider.urlBuilder(UrlPath.CustomDev)],
+		[arrows.nth(1), UrlProvider.urlBuilder(UrlPath.MobileDev)],
+		[arrows.nth(2), UrlProvider.urlBuilder(UrlPath.CloudDevelopment)],
+		[arrows.nth(3), UrlProvider.urlBuilder(UrlPath.DevOpsServ)],
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/mainSite/services/digitalTransformation/DigitalTransformationActions.test.ts:67]: DevOps section without an arrow

File: ui_tests_playwright/automation/test/ui/mainSite/services/digitalTransformation/DigitalTransformationActions.test.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+		[arrows.nth(4), UrlProvider.urlBuilder(UrlPath.BigData)],
+		[arrows.nth(5), UrlProvider.urlBuilder(UrlPath.InternetOfThings)],
+		[arrows.nth(5), UrlProvider.urlBuilder(UrlPath.AiDevelopment)],
+		[arrows.nth(5), UrlProvider.urlBuilder(UrlPath.QaAsAServ)],
+		[arrows.nth(5), UrlProvider.urlBuilder(UrlPath.UiUxDesign)],
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/mainSite/services/digitalTransformation/DigitalTransformationActions.test.ts:72]: numbers

File: ui_tests_playwright/automation/test/ui/mainSite/services/digitalTransformation/DigitalTransformationActions.test.ts [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+	]);
+
+	for (const [arrow, url] of arrowUrlMap) {
+		await baseDriverSteps.checkRedirectToPage(arrow, url, UrlProvider.urlBuilder(UrlPath.DigitalTransform));
+	}
+});
+
+test('Check redirects by "Clutch Review" button in "Digital Transformation Strategy" container from the "Digital Transformation" page @Regression @DigitalTransformation @TSWEB-1135', async () => {
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/mainSite/services/digitalTransformation/DigitalTransformationActions.test.ts:80]: mb redirect?

File: ui_tests_playwright/automation/test/ui/mainSite/services/digitalTransformation/DigitalTransformationActions.test.ts [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+	const digitalStrategyContainer = driver.getByTestId(DigitalTransformation.DigitalTransformationStrategy);
+	const clutchReviewButtons = digitalStrategyContainer.getByTestId(Buttons.Clutch);
+
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/mainSite/services/digitalTransformation/DigitalTransformationActions.test.ts:83]: mb clutchReviewButton

**Review by Valerii-Kryvokhyzha — CHANGES_REQUESTED** [review]

**Review by Valerii-Kryvokhyzha — APPROVED** [review]

**Review by anastasiia-liashova — APPROVED** [review]

**Review by DaniilVostrikov — APPROVED** [review]

**Pull request merged**